arkouda/pdarrayclass.py

```diff
@@ -707,7 +899,7 @@ def save(self, prefix_path, dataset='array', mode='truncate') -> str:
 #   only after:
 #       all values have been checked by python module and...
 #       server has created pdarray already befroe this is called
-def create_pdarray(reqMsg : str) -> pdarray:
+def create_pdarray(reqMsg : str) -> 'pdarray':
```

Comment:
the input argument should be called `repMsg`(response message) and not `reqMsg`(request message) maybe this has been this way forever ;-) just to be clear it's wrong in the whole function so it operates correctly. It's just a variable naming thing

Reply:
fixed!